src/pages/ProfilesExplorerView/components/SceneExploreFavorites/SceneExploreFavorites.tsx

```diff
@@ -102,25 +100,19 @@ export class SceneExploreFavorites extends SceneObjectBase<SceneExploreFavorites
   }
 
   openExpandedPanelDrawer(item: GridItemData) {
+    const headerActions = () => [
+      new SelectAction({ EventClass: EventViewServiceLabels, item }),
+      new SelectAction({ EventClass: EventViewServiceFlameGraph, item }),
+    ];
+
     this.state.drawer.open({
       title: item.label,
-      body:
-        item.panelType === PanelType.BARGAUGE
-          ? new SceneLabelValuesBarGauge({
-              item,
-              headerActions: () => [
-                new SelectAction({ EventClass: EventViewServiceLabels, item }),
-                new SelectAction({ EventClass: EventViewServiceFlameGraph, item }),
-              ],
-            })
-          : new SceneLabelValuesTimeseries({
-              displayAllValues: true,
-              item,
-              headerActions: () => [
-                new SelectAction({ EventClass: EventViewServiceLabels, item }),
-                new SelectAction({ EventClass: EventViewServiceFlameGraph, item }),
-              ],
-            }),
+      body: panelBuilder(item.panelType, {
+        displayAllValues: true,
+        legendPlacement: 'right',
```

Comment:
Moving the placement of the legend when opened in the drawer (we have more width).